packages/ui/src/components/organisms/PaymentMethods/styles.scss

```diff
@@ -44,17 +44,15 @@
     justify-content: space-between;
     margin-top: var(--fs-payment-methods-flags-margin-top);
 
```

Comment:
question: Are we keeping the transparent background for this component in footer?

|before|now|
|-|-|
|<img width="331" alt="image" src="https://github.com/user-attachments/assets/3b6577d0-6460-4945-9d5c-7604f7b77859">|<img width="304" alt="image" src="https://github.com/user-attachments/assets/7d46e463-01f5-4770-86fa-6a4410387a32">

Reply:
Fan, added this background-color as a new token: https://github.com/vtex/faststore/pull/2120/commits/a08c0f2f2f59bc5f6b4a550e35e95b6acd9eb0be

Reply:
nice! ✅